{{ cookiecutter.project_name }}/pyproject.toml

```
@@ -35,12 +35,30 @@ include = ["{{ cookiecutter.package_name }}*"]
 [tool.setuptools.dynamic]
 version = {attr = "{{ cookiecutter.package_name }}.__version__"}
 
-[tool.isort]
-profile = "black"
-force_alphabetical_sort_within_sections = true
-force_single_line = true
-
 [tool.pytest.ini_options]
-norecursedirs=".venv data doc etc *.egg-info misc var build lib include"
+norecursedirs="venv .venv data doc etc *.egg-info misc var build lib include"
 python_files = "test_*.py"
 testpaths = "{{ cookiecutter.package_name }}"
+
+[tool.zestreleaser]
```

```suggestion
[tool.zest-releaser]
```
as per https://zestreleaser.readthedocs.io/en/latest/options.html#per-project-options